pet-kata/src/test/java/org/eclipse/collections/petkata/Exercise5Test.java

```diff
@@ -84,9 +84,8 @@ public void refactorToEclipseCollections()
             }
         }
 
-        Assert.assertEquals(Sets.mutable.with(1, 2, 3, 4), petAges);
         //extra bonus - convert to a primitive collection
-        Assert.assertEquals(IntSets.mutable.with(1, 2, 3, 4), petAges);
+        Assert.assertEquals(Sets.mutable.with(1, 2, 3, 4), petAges);
```

Comment:
It will be confusing to users to see the same Assert.equals() statement on two lines. We can make it easier to understand by doing this: `Assert.assertEquals("Extra Credit - convert to a primitive collection", Sets.mutable.with(1, 2, 3, 4), petAges);`
I will merge it once your IP validation is in, and once you fix this to include a comment.